AnagramNew/AnagramNew.java

```diff
@@ -0,0 +1,42 @@
+
+
+import java.util.*;
+
+public class AnagramNew {
+
+    static String[] dictionary = {"anna", "ivan", "naan", "vani", "pyotr", "nana", "navi"};
+
+    public static void main(String[] args) {
+        AnagramNew a = new AnagramNew();
```

Comment:
As far as I remember, the task has been formulated like we need a method, which accepts a string word and returns all its anagrams. I do not see such method in the class.